lib/language.js

```diff
@@ -22,13 +22,14 @@ function init() {
     , { code: 'pl', language: 'Polski' }
     , { code: 'pt', language: 'Português (Brasil)' }
     , { code: 'ro', language: 'Română' }
+    , { code: 'ru', language: 'Русский' }
     , { code: 'sv', language: 'Svenska' }
     , { code: 'fi', language: 'Suomi' }
   ];
 
   var translations = {
     // Server
-    'Listening on port' : {
+    'Listening on port--' : {-
       cs: 'Poslouchám na portu'
       ,es: 'Escuchando en el puerto'
```

Comment:
Hi, Jason

Sorry know nothing about programming 

Tried doing everything by the instruction

Will correct the issues when I get back to the translation.

Hope won’t create any BIG problems. 

Otherwise let me know and I’ll get off.

Fedor

 

From: Jason Calabrese [mailto:notifications@github.com] 
Sent: Sunday, March 13, 2016 11:16 PM
To: nightscout/cgm-remote-monitor <cgm-remote-monitor@noreply.github.com>
Cc: apanasef <apanasef@gmail.com>
Subject: Re: [cgm-remote-monitor] Russian language unfinished v0.1 (#1548)

 

In  <https://github.com/nightscout/cgm-remote-monitor/pull/1548#discussion_r55942948> lib/language.js:

>      , { code: 'sv', language: 'Svenska' }
>      , { code: 'fi', language: 'Suomi' }
>    ];
>  
>    var translations = {
>      // Server
> -    'Listening on port' : {
> +    'Listening on port--' : {-
>        cs: 'Poslouchám na portu'
>        ,es: 'Escuchando en el puerto'

 <https://camo.githubusercontent.com/37ddd55708a38212f4e9de4e3eac98358fefe597/68747470733a2f2f7777772e636f646163792e636f6d2f6173736574732f696d616765732f66617669636f6e2e706e67> Issue found:  <https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2232517981/issues/source?bid=2452379&fileBranchId=3176126#l34> Expected an assignment or function call and instead saw an expression.

—
Reply to this email directly or  <https://github.com/nightscout/cgm-remote-monitor/pull/1548/files#r55942948> view it on GitHub.